.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish Packages
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Bootstrap Lerna
+        run: npx lerna bootstrap
+
+      - name: Publish packages
+        run: npx lerna publish from-package --no-private
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Don't we also need a ` NPM_TOKEN` to publish on npm ? 
I believe `GITHUB_TOKEN` are only meant for github actions such as clone, push or publish on [Github-packages](https://docs.github.com/fr/packages/learn-github-packages/introduction-to-github-packages).
Isn't our goal to publish on npmjs ?